movies/src/components/addMovie/AddMovie.js

```diff
@@ -1,19 +1,40 @@
 import React, { useState } from 'react';
 import PropTypes from 'prop-types';
+import {bindActionCreators} from "redux";
+import {connect} from "react-redux";
 import Input from '../common/input/Input';
 import Button from '../common/button/Button';
 import MultiSelectInput from '../common/multiSelectInput/MultiSelectInput';
 import Datepicker from '../common/datepicker/Datepicker';
 import CloseButton from '../closeButton/CloseButton';
 import MovieActionPopup from '../movieActionPopup/MovieActionPopup';
 import Constants from '../constants';
+import {editMoviesList} from "../../store/actions/actions";
 
-export default function AddMovie(props) {
+const matchDispatchToProps = (dispatch) => {
+  return bindActionCreators({updateMovies: editMoviesList}, dispatch)
+};
+
+const mapStateToProps = (state) => {
+  return {
+    state,
```

Comment:
this will cause `AddMovie` to rerender each time store is changed . usually, in React we pass only required slices of data (in your case it will be `movies`)

Reply:
Additionally, could you consider using `useSelector` hook instead of `connect` HOC?